### DISCUSSION TITLE
Yellow dot of warnings is not visible

### DISCUSSION THREAD

**Discussion by LukasKarel** [datalust/seq-tickets#2241] [Help] [answered]
In version 2024.3.12021 of seq the yellow dot of warnings is not visible anymore. Debug (grey dot) and Errors (red dot) are still working as expected. Is there anything we have to configure to get it back or is this a bug?

**Comment by nblumhardt** [comment]
Hi Lukas, thanks for getting in touch. This seems to be working for me - if you expand a warning event, what text do you see in the `Level` menu item? (E.g. `Level (Warning)` or something similar? Thanks!

**Comment by LukasKarel** [comment]
Thanks for the fast reply!
It says `Level (Warn)`.

**Reply by nblumhardt** [reply, marked as answer]
This looks like a regression; I've raised https://github.com/datalust/seq-tickets/issues/2242 and we'll address this in the next patch. Thanks for letting us know :+1:
